frontend/src/domain/use_cases/layer/administrative/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function getLayerNameFromTypeAndZone(type: string, zone: string | undefined) {
```

Comment:
Same thing. It's obvious we have a string in return here but it's nice to be explicit (both for params & return) for functions that are exported.